.ansible-lint

```diff
@@ -1,6 +1,9 @@
 skip_list:
   - yaml
-  - fqcn-builtins  # Use FQCN for builtin actions.
-  - no-handler
-  - command-instead-of-shell
-  - no-jinja-when
+  - 'fqcn[action-core]'
+  - 'fqcn[action]'
+  - 'no-handler'
+  - 'command-instead-of-shell'
+  - 'no-jinja-when'
```

Comment:
I didn't know, it was even possible to break that rule. I thought, Ansible would fail. Ok, just learned something new, thanks.

Please resolve this thread when you read it.